shared/chat/inbox/row/big-team-header/index.js

```diff
@@ -80,6 +80,7 @@ const styles = styleSheetCreate({
   },
   team: platformStyles({
     common: {
+      color: globalColors.black_60,
```

Comment:
I prefer `black_75`, but it's not a deal breaker.

Reply:
I like it. `black_60` is BodySmallSemiboldSecondaryLink and consistent with our clickable section labels.